src/Application/Actions/User/UserAction.php

```diff
@@ -6,13 +6,13 @@
 
 use Psr\Log\LoggerInterface;
 use BigGive\Identity\Application\Actions\Action;
-use BigGive\Identity\Repository\UserRepository;
+use BigGive\Identity\Repository\PersonRepository;
 
 abstract class UserAction extends Action
 {
-    protected UserRepository $userRepository;
+    protected PersonRepository $userRepository;
```

Comment:
@NoelLH it's probably wiser to rename the variable too from `$userRepository` to `$personRepository`

Reply:
@alihejazi yep, this was an IDE auto rename. I forgot about this whole folder which was in the skeleton and don't think we'll use it, so will delete entirely. Our Actions will be all JSON API if they return stuff so I think `Get` is a more helpful verb if we bring a similar action back.